docs/release/Betaflight-4.5-Release-Notes.md

```diff
@@ -96,31 +98,60 @@ The `MAG_CALIB` and `MAG_TASK_RATE` debugs have been added to investigate calibr
 
 Please read the [wiki note](https://betaflight.com/docs/wiki/archive/Magnetometer) carefully, and test it thoroughly, before using the Mag in a GPS Rescue.  Note that the current default for GPS Rescue is to use the Mag.  If you are not 100% sure that your Mag is working, don't use it.
 
-thanks to pichim, ctzsnooze, SteveCEvans, ledvinap
+Thanks to: pichim, ctzsnooze, SteveCEvans, ledvinap
+
+## 5. Mapping of GPS flights within Blackbox Explorer and with Export GPX
+
+Two awesome mapping features for GPS users who have made a log file that contains GPS data.
+
+`Export GPX` enables the `Export GPX` button at the top of the BBE window when a log file with GPS data is opened.  The exported `.gpx` file can be imported into online mapping software, such as [gpxStudio](https://gpx.studio), drawing your flights over a map.
+
+Detailed explanatory video [here](https://www.youtube.com/watch?v=dhgQ8aPUq_U).
+For more information see [PR 614](https://github.com/betaflight/blackbox-log-viewer/pull/614)
+
+`Within BBE GPS Mapping` shows a smaller, simple map within Blackbox explorer itself.  To open the map, user only has to click the view/hide map icon from the `Overlay` group of menu icons (at the top of the normal blackbox screen).
+
+The position of the quad moves over the map as the cursor moves through the file.  The colour trail can show altitude (see the 'cog' for BBE settings to configure this).
+
+For more information see [PR 613](https://github.com/betaflight/blackbox-log-viewer/pull/613)
+
+Thanks for both to: bonchan
+
+## 6. Support for colour fonts if supported by the HD VTX
+
+White, green, orange or red colours can now be used for text and symbols are now supported for compatible HD Vtx modules, eg Walksnail.
+
+See: [13005](https://github.com/betaflight/betaflight/pull/13005)
```

Comment:
Can I get a "PR" for consistency?

Reply:
indeed, and spaces now.  thanks for the careful review!